src/main/java/bio/terra/service/dataset/dao/DatasetTableDao.java

```diff
@@ -43,6 +45,7 @@
     private static final String sqlSelectColumn = "SELECT id, name, type, array_of FROM dataset_column " +
         "WHERE table_id = :table_id";
 
+    // TODO: get rid of this connection usage
     private final DataSource jdbcDataSource;
     private final NamedParameterJdbcTemplate jdbcTemplate;
     private final ObjectMapper objectMapper;
```

Comment:
Also, any reason we're not just autowiring all of these objects and using a constructor?